.github/workflows/pr_test_build.yml

```diff
@@ -111,6 +114,16 @@ jobs:
       - name: Send Test APK
         run: |
           cd /opt/android/cake_wallet
-          var=$(curl --upload-file build/app/outputs/apk/release/app-release.apk https://transfer.sh/app-release.apk)
+          var=$(curl --upload-file build/app/outputs/apk/release/app-release.apk https://transfer.sh/$GITHUB_HEAD_REF.apk -H "Max-Days: 10")
           curl ${{ secrets.SLACK_WEB_HOOK }} -H "Content-Type: application/json" -d '{"apk_link": "'"$var"'","ticket": "'"$GITHUB_HEAD_REF"'"}'
 
+      - name: Rename apk file
+        run: |
+          cd /opt/android/cake_wallet/build/app/outputs/apk/release
+          mkdir test-apk
+          mv app-release.apk test-apk/$GITHUB_HEAD_REF.apk
+
+      - name: Upload Artifact
+        uses: kittaakos/upload-artifact-as-is@v0
```

Comment:
Why is this used instead of `actions/upload-artifact`?

Reply:
I tried both, but this will upload the artifact with the given file name `actions/upload-artifact` had an issue with making the artifact name with a variable name and required more steps, but they are both the same, this one is even built based on `actions/upload-artifact`